src/betterproto/plugin/typing_compiler.py

```diff
@@ -139,28 +139,32 @@ def imports(self) -> Dict[str, Optional[Set[str]]]:
 class NoTyping310TypingCompiler(TypingCompiler):
     _imports: Dict[str, Set[str]] = field(default_factory=lambda: defaultdict(set))
 
+    @staticmethod
+    def _fmt(type: str) -> str:  # for now this is necessary till 3.14
+        return type.removeprefix('"').removesuffix('"')
+
     def optional(self, type: str) -> str:
-        return f"{type} | None"
+        return f'"{self._fmt(type)} | None"'
```

Comment:
```suggestion
        return f"{self._fmt(type)} | None"
```

Doing this for all of the return values you changed in this file will fix the CI issues.

Reply:
That's not the fix here unfortunately. The tests actually need updating

Reply:
Sad :(

Reply:
(Also my Python knowledge is definitely not library level, so I defer to you to know the best approach to handle the fix)